.github/workflows/notify.yml

```diff
@@ -0,0 +1,29 @@
+name: notify
+on:
+  check_suite:
+    types: [completed]
+
+jobs:
+  notify-master:
+    runs-on: ubuntu-latest
+    if: github.event.check_suite.conclusion != 'success' && github.event.check_suite.head_branch == 'master'
+    steps:
+      - name: Install gh-pr-comment
+        uses: at-wat/setup-gh-pr-comment@v0
+      - name: Get PR number
+        id: pr_number
+        uses: at-wat/get-pr-number-action@support-check-suite-event-released
```

Comment:
will be changed to `kamatama41/get-pr-number-action@v0` once https://github.com/kamatama41/get-pr-number-action/pull/31 is released